acceptance-tests/src/acceptance-test/java/tech/pegasys/teku/test/acceptance/das/DasSyncAcceptanceTest.java

```diff
@@ -0,0 +1,122 @@
+    final TekuBeaconNode primaryNode =
+        createTekuBeaconNode(
+            createConfigBuilder()
+                .withRealNetwork()
+                .withDasExtraCustodyGroupCount(subnetCount - defaultCustodySubnetCount)
+                .build());
+
+    primaryNode.start();
+    UInt64 genesisTime = primaryNode.getGenesisTime();
+    final TekuBeaconNode lateJoiningNode =
+        createLateJoiningNode(primaryNode, genesisTime.intValue());
+    primaryNode.waitForEpochAtOrAbove(fuluEpoch + 1, Duration.ofMinutes(5));
+
+    lateJoiningNode.start();
+    lateJoiningNode.waitForGenesis();
+    lateJoiningNode.waitUntilInSyncWith(primaryNode);
+    lateJoiningNode.waitForEpochAtOrAbove(6, Duration.ofMinutes(5));
+    lateJoiningNode.waitUntilInSyncWith(primaryNode);
+
+    int epochSlots = primaryNode.getSpec().slotsPerEpoch(UInt64.ZERO);
+    int endSlot = 6 * epochSlots;
+    int firstFuluSlot = fuluEpoch * epochSlots;
+    int totalColumns =
+        IntStream.range(firstFuluSlot, endSlot)
+            .mapToObj(UInt64::valueOf)
+            .map(slot -> getAndAssertDasCustody(lateJoiningNode, slot))
+            .mapToInt(i -> i)
+            .sum();
+
+    assertThat(totalColumns).isGreaterThan(0);
+  }
+
+  private int getAndAssertDasCustody(final TekuBeaconNode node, final UInt64 fuluSlot) {
+    try {
+      Optional<SignedBeaconBlock> maybeBlock = node.getBlockAtSlot(fuluSlot);
+      if (maybeBlock.isPresent()) {
+        SignedBeaconBlock block = maybeBlock.get();
+        boolean hasBlobs =
+            !block
+                .getBeaconBlock()
+                .orElseThrow()
+                .getBody()
+                .toVersionDeneb()
+                .orElseThrow()
+                .getBlobKzgCommitments()
+                .isEmpty();
+        int columnCount = node.getDataColumnSidecarCount(block.getRoot().toHexString());
+        if (hasBlobs) {
+          assertThat(columnCount).isNotZero();
+        } else {
+          assertThat(columnCount).isZero();
+        }
+        return columnCount;
+      } else {
+        return 0;
+      }
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
+  }
+
+  private TekuBeaconNode createLateJoiningNode(
+      final TekuBeaconNode primaryNode, final int genesisTime) throws IOException {
+    return createTekuBeaconNode(
+        createConfigBuilder()
+            .withGenesisTime(genesisTime)
+            .withRealNetwork()
+            .withPeers(primaryNode)
+            .withInteropValidators(0, 0)
+            .build());
+  }
+
+  private TekuNodeConfigBuilder createConfigBuilder() throws IOException {
+    return TekuNodeConfigBuilder.createBeaconNode()
+        .withNetwork("minimal")
+        .withAltairEpoch(UInt64.valueOf(0))
+        .withBellatrixEpoch(UInt64.valueOf(0))
+        .withCapellaEpoch(UInt64.valueOf(0))
+        .withDenebEpoch(UInt64.valueOf(0))
+        .withElectraEpoch(UInt64.valueOf(1))
```

Comment:
We could move electra to 0, fulu to 1 and shave 1 epoch from total to reduce test time

Reply:
> We could move electra to 0, fulu to 1 and shave 1 epoch from total to reduce test time

Yep, thanks that worked! I have also reduced the final test epoch, and the test is running around 3:15 now

Reply:
Generally... with the `swift` config which has `slot: 3s, epoch: 4 slots` the GitHub runner (likely quite weak vps) struggles heavily and there are lots of empty slots. This is likely because of heavy KZG operations. 
That's why `swift` has been changed to `minimal` (`slot: 6s, epoch: 8 slots`)
Probably teku repo CI has better runner instances and `swift` config would work fine there. 

Does it make sense to implement some stub kzg and enable it (together with stub BLS) for acceptance tests? The full stack could be tested with a separate simpler and shorter tests, while complex scenarios may utilize stub implementations.

Reply:
Got it. Let's keep it with real kzg, we could return to this option if it will be an issue again